### DISCUSSION TITLE
Missing feature: External image editor (suggestion #122)

### DISCUSSION THREAD

**Discussion by jacobtriffo** [d2phap/ImageGlass#398] [General] [answered]
### System information:
 - Windows OS version: 1803 build 17134.288
 - ImageGlass version: 5.5.7.26 and Moon_5.1.5.24

### Describe the bug:
The option to select an external image editor is unavailable, though pull request https://github.com/d2phap/ImageGlass/pull/175 implies that it has been added as of 4.0

### Expected behavior:
External editor should be available in settings, but is not. 

### To Reproduce
1. Open image in image glass
2. right click image and edit
3. close mspaint because it's useless
4. dig through settings and not find an option to change this behavior

**Comment by d2phap** [comment, marked as answer]
Hi @jacobtriffo 

This feature was implemented since v4.0
To change the editor for specific image format:
- Open *Settings* > *Image* tab
- Scroll to *Image Editing* section
- Select a format and click *Edit* button

![image](https://user-images.githubusercontent.com/3154213/44250775-63b50780-a228-11e8-8ee8-6ee1bbc9b09f.png)

**Comment by jacobtriffo** [comment]
I'm running 7.5.1.1 (current) and my image tab in settings does not have an 'image editing' section. 

For anyone who finds this, they've moved it to the 'edit' tab of the settings menu.

**Reply by d2phap** [reply]
thanks @jacobtriffo for the update!
